.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: publish package on pypi
+
+on:
+  release:
+    types: [published, edited]
```

Comment:
Why on edited?

Reply:
No reason really - I can remove it. Think I added it to a project when the initial release failed because of bad credentials in the repo.

Pypi won't allow you to re-republish a package though, so you could in theory run the workflow 100x and it would only publish successfully once.

Reply:
I assume `edited` will be triggered also with some simple typo fix? Let's have only `published` here 🙂